examples/speaker_tasks/diarization/asr_with_diarization.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import argparse
+
+from nemo.collections.asr.parts.utils.diarization_utils import (
+    ASR_DIAR_OFFLINE,
+    get_file_lists,
+)
+from nemo.utils import logging
+
+"""
+Supported ASR models: QuartzNet15x5Base
+
+CH109: All sessions have two speakers.
+
+python get_json_ASR_and_diarization.py \
+--audiofile_list_path='/disk2/scps/audio_scps/callhome_ch109.scp' \
+--reference_rttmfile_list_path='/disk2/scps/rttm_scps/callhome_ch109.rttm' \
+--oracle_num_speakers=2
+
+AMI: Oracle number of speakers in EN2002c.Mix-Lapel is 3, not 4.
+
+python get_json_ASR_and_diarization.py \
+--audiofile_list_path='/disk2/datasets/amicorpus/mixheadset_test_wav.list' \
+--reference_rttmfile_list_path='/disk2/datasets/amicorpus/mixheadset_test_rttm.list' \
+--oracle_num_speakers=2
```

Comment:
Can we replace this with general comments replacing our directory structure?

Reply:
I removed these since these are in the README.md.